config.js

```diff
@@ -23,5 +27,18 @@ module.exports = {
     secretAccessKey: process.env.AWS_PASSWORD || '',
     transportType: 'ses',
     region: process.env.EMAIL_REGION || ''
-  }
+  },
+  upload: {
+    maxFileSize: '100mb',
+    hostname: env !== 'production' ?
+      `http://${localhost()}/api/image-upload` :
+      process.env.FILE_VAULT_URL
```

Comment:
Same comment as above - can we use `FILE_VAULT_URL` as preference whenever specified and only infer from `NODE_ENV` as a fallback?